pkg/apiserver/archive/archive.go

```diff
@@ -0,0 +1,56 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package archive
+
+import (
+	"github.com/gin-gonic/gin"
+
+	"github.com/pingcap/chaos-mesh/pkg/core"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/pingcap/chaos-mesh/pkg/config"
+)
+
+// Service defines a handler service for archive experiments.
```

Comment:
Is archive also like events in the database? Can I ask what's the difference between archive and event?

Reply:
This archive just defines for the experiment.  Maybe it ’s better to name it to "experiment".